src/urllib3/connection.py

```diff
@@ -482,6 +482,7 @@ def getresponse(  # type: ignore[override]
             headers=headers,
             status=httplib_response.status,
             version=httplib_response.version,
+            version_string=getattr(self, "_http_vsn_str", "HTTP/?"),
```

Comment:
Emscripten has its own connection class, so I don't need the getattr any more:

```suggestion
            version_string=self._http_vsn_str,
```

Reply:
But I do need to add `version_string` to https://github.com/urllib3/urllib3/blob/2aec09f6a1727dd54f65b9069ef8f168ef40a885/src/urllib3/contrib/emscripten/response.py#L47. Hopefully CI will catch this (did not run so far).